docker/php-fpm/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM php:8.2.10-fpm-alpine
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN set -eux \
+    && apk add --no-cache \
+        c-client \
+        ca-certificates \
+        freetds \
+        freetype \
+        gettext \
+        gmp \
+        icu-libs \
+        imap \
+        libffi \
+        libgmpxx \
+        libintl \
+        libjpeg-turbo \
+        libpng \
+        libpq \
+        libssh2 \
+        libstdc++ \
+        libtool \
+        libxpm \
+        libxslt \
+        libzip \
+        make \
+        sqlite \
+        tidyhtml \
+        tzdata \
+        unixodbc \
+        vips \
+        yaml
+
+RUN set -eux \
```

Comment:
@mwarzybok-sumoheavy each `RUN` command creates a new Docker layer. Unless necessary, we should combine.